src/app/app.constants.ts

```diff
@@ -70,7 +70,7 @@ const applicationUserThemes: Theme[] =  [
   },
   {
     roles: ['pui-case-manager'],
-    appTitle: {name: 'Manage Cases', url: '/'},
+    appTitle: {name: 'Manage cases', url: '/'},
```

Comment:
The application is called Manage case, so the url is https://manage-case.platform.hmcts.net so can we please change it to "Manage case"?